evap/staff/views.py

```diff
@@ -764,9 +764,21 @@ def course_comments(request, semester_id, course_id):
         section_list = course_sections if contribution.is_general else contributor_sections
         section_list.append(CommentSection(questionnaire, contribution.contributor, contribution.label, contribution.responsible, text_results))
 
-    template_data = dict(semester=semester, course=course, course_sections=course_sections,
-            contributor_sections=contributor_sections, filter_comments=filter_comments)
-    return render(request, "staff_course_comments.html", template_data)
+    template_data = dict(semester=semester, course=course, view=view)
+
+    if view == 'quick':
+        next_course = semester.course_set.annotate(num_unreviewed_textanswers=Count("contributions__textanswer_set", filter=Q(contributions__textanswer_set__state=TextAnswer.NOT_REVIEWED), distinct=True)) \
+            .exclude(pk=course.pk) \
+            .exclude(state='published') \
+            .filter(num_unreviewed_textanswers__gt=0) \
```

Comment:
i would put the annotate behind the excludes, because semantically you don't want to annotate the whole `course_set`.